chức năng đăng nhập

minhnk1995 opened the pull request:
-tạo màn hình đăng nhập
-validate login
-thêm filter login

minhnk1995 committed:
chức năng đăng nhập

-tạo màn hình đăng nhập
-validate login
-thêm filter login

devhuunhanhcm commented on src/main/java/cybersoft/javabackend/crm/repository/UserRepository.java:
```diff
@@ -2,16 +2,63 @@
+				User user = new User();
+				user.setEmail(res.getString("email"));
+				user.setPassword(res.getString("password"));
+				user.setName(res.getString("fullname"));
```
thêm attribute phone nữa anh zai ơi

devhuunhanhcm committed:
test

minhnk1995 committed:
update chức năng  đăng nhập

-sửa query  hàm findAll()
-xóa hàm getUserRole()
-thêm chức năng nhớ thông tin đăng nhập

minhnk1995 committed:
Update login.jsp

minhnk1995 committed:
Delete img.jpg
